test/system/cli-entrypoint-create-app.test.ts

```diff
@@ -0,0 +1,54 @@
+import { introspectionQuery } from 'graphql'
+import { setupE2EContext } from '../../src/utils/e2e-testing'
+import { rootLogger } from '../../src/utils/logger'
+
+rootLogger.settings({ level: 'trace' })
+const log = rootLogger.child('system-test')
+
+const ctx = setupE2EContext({
+  linkedPackageMode: true,
+})
+
+test('cli entrypoint create app', async () => {
```

Comment:
A lot of this test is repeated from the e2e variant. We can probably refactor. It will be especially relevant to do so once we add many more test variations, like `npm+yarn` and `database+no-database`, ... so on.